components/ui/list/index.tsx

```
@@ -1,11 +1,68 @@
+
+  // Use window scrolling when available to avoid nested scrollbars
+  useEffect(() => {
+    setScrollElement(typeof window !== "undefined" ? window.document.scrollingElement : null);
+  }, []);
+
+  // Use virtualization for larger lists only
+  const shouldVirtualize = (events?.length || 0) > 24;
+
+  const rowVirtualizer = useVirtualizer({
+    count: events?.length || 0,
+    getScrollElement: () => (shouldVirtualize ? (scrollElement as any) : null),
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)